README.md

```diff
@@ -1,53 +1,41 @@
 # Project Name
 
-(short, 1-3 sentenced, description of the project)
+ACS Calling Sample for WPF.
```

Comment:
It would be better informing in the description that this sample app uses WebView2 and the Web SKU of the Calling SDK.